tpm2/tpm2_test.go

```diff
@@ -675,3 +675,34 @@ func TestEncodeDecodeCreationAttestationData(t *testing.T) {
 		t.Errorf("got decoded value:\n%v\nwant:\n%v", decoded, ad)
 	}
 }
+
+func TestEncodeDecodeIdenticalRSADefaultExponent(t *testing.T) {
+	p := Public{
+		Type:       AlgRSA,
+		NameAlg:    AlgSHA1,
+		Attributes: FlagSign | FlagSensitiveDataOrigin | FlagUserWithAuth,
+		RSAParameters: &RSAParams{
+			Sign: &SigScheme{
+				Alg:  AlgRSASSA,
+				Hash: AlgSHA1,
+			},
+			KeyBits:                     2048,
+			EncodeDefaultExponentAsZero: true,
+			Exponent:                    defaultRSAExponent,
```

Comment:
Add a test for `EncodeDefaultExponentAsZero==false` (within the same test func).